charts/atlantis/templates/secret-webhook.yaml

```diff
@@ -10,7 +10,7 @@ data:
   key.pem: {{ required "githubApp.key is required if githubApp configuration is specified." .Values.githubApp.key | b64enc }}
   github_secret: {{ required "githubApp.secret is required if githubApp configuration is specified." .Values.githubApp.secret | b64enc }}
   {{- end}}
-  {{- if .Values.github.user }}
```

Comment:
While this makes sense, I would change this on a different PR. Also since there are other places where this can be improved ([example](https://github.com/runatlantis/helm-charts/blob/main/charts/atlantis/templates/statefulset.yaml#L296)).

Reply:
Removed it will be taken in another pr